ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/sync/tasks/DetermineCommonAncestorTaskTest.java

```diff
@@ -57,57 +58,34 @@
 
 public class DetermineCommonAncestorTaskTest {
```

Comment:
This file looks pretty messy, but in summary:  1 test was added for the bug fix, 2 tests for undefined behavior were removed, the generic param on this task was removed, and otherwise there's just a little bit of cleanup / reorganization.